x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/expression/gen/processor/ConstantProcessorTests.java

```diff
@@ -28,7 +31,10 @@ protected ConstantProcessor createTestInstance() {
 
     @Override
     protected ConstantProcessor mutateInstance(ConstantProcessor instance) throws IOException {
-        return new ConstantProcessor(randomValueOtherThan(instance.process(null), () -> randomAlphaOfLength(5)));
+        return new ConstantProcessor(randomValueOtherThan(instance.process(null),
```

Comment:
Why not also keeping the original `randomAlphaOfLength`? Using a `randomBoolean` you could switch between `randomAlphaOfLength` and an `IntervalDayTime` instance.

Reply:
Because the randomAlpha is already tested from the initial instance which gets serialized/deserialized so using it again wouldn't change much.